src/main/java/seedu/duke/ui/Formatter.java

```diff
@@ -31,16 +33,47 @@
     /** Length of a ansi defined color. */
     private static final int ANSI_PREFIX_LENGTH = 5;
 
-    public static String formatNotebook(String header, Notebook notebook) {
+    /**
+     * Method compiles the ArrayList items and appends the items to a String.
+     *
+     * @param notes ArrayList of notes to obtain note title/tags from
+     * @return noteString StringBuilder containing the notes ready to be printed
+     */
+    public static String formatNotes(String header, ArrayList<Note> notes) {
         String formattedString = "";
-        return formattedString;
+        return getNoteString(header, notes, formattedString);
     }
 
-    public static String formatNotebook(Notebook notebook) {
+    /**
+     * Method compiles the ArrayList items and appends the items to a String.
+     *
+     * @return noteString StringBuilder containing the notes ready to be printed
+     */
+    public static String formatNotes(String pinnedHeader, String unpinnedHeader,
+                                     ArrayList<Note> pinned, ArrayList<Note> unpinned) {
         String formattedString = "";
+        formattedString = getNoteString(pinnedHeader, pinned, formattedString);
+        formattedString = getNoteString(unpinnedHeader, unpinned, formattedString);
```

Comment:
can remove the last parameter for the function. Suggested way:

formattedString = getNoteString(pinnedHeader, pinned);
formattedString = formattedString.concat(getNoteString(unpinnedHeader, unpinned)